collection-spec/json-schema/collection.json

```diff
@@ -94,6 +114,27 @@
           "type": "string"
         }
       }
+    },
+    "aggregation": {
+      "type": "object",
+      "required": ["agg_type"],
+      "properties": {
+        "agg_type": {
+          "title": "Type of aggegration",
+          "description": "Type of aggegation operation to apply: join_new, join_existing, union",
```

Comment:
You probably want to use an enum here.
https://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values

Reply:
yeah! that's what I wanted, but didn't know how to do it.